chunky/src/java/se/llbit/chunky/ui/render/AdvancedTab.java

```diff
@@ -192,6 +183,14 @@ public PictureExportFormat fromString(String string) {
       scene.setPreventNormalEmitterWithSampling(newvalue);
       PersistentSettings.setPreventNormalEmitterWithSampling(newvalue);
     });
+
+    rendererSelect.setTooltip(new Tooltip("The renderer to use on a final render."));
```

Comment:
```suggestion
    rendererSelect.setTooltip(new Tooltip("The renderer to use for rendering."));
```

(although that sounds strange too)